eclipse-bluechi/README.md

```diff
@@ -0,0 +1,206 @@
+    code .
+    ```
+
+1. VSCode detects automatically that a `.devcontainer` folder exists inside this subfolder. Please confirm the dialog to reopen VSCode inside the devcontainer. Afterwards, open a new terminal inside the devcontainer in VSCode.
+
+### Devcontainers CLI
+
+#### Prerequisites
+
+Ensure you have the `devcontainer` cli installed: https://github.com/devcontainers/cli?tab=readme-ov-file#try-it-out
+
+#### Instructions
+
+Run the following from `<absolute/path/to>/software-orchestration/eclipse-bluechi`:
+
+```sh
+devcontainer up \
+--workspace-dir .
+```
+
+You will need to specify the `--docker-path` option if you are using Podman:
+
+```sh
+devcontainer up \
+--workspace-dir . \
+--docker-patch /usr/bin/podman
+```
+
+You can now enter into the devcontainer and iteract with Eclipse BlueChi:
+
+```sh
+docker exec -it $container_id_or_name /bin/bash
+bluechictl list-units
+```
+
+### Docker/Podman
+
+You can also directly run this blueprint directly with Docker or Podman.
+
+Start the blueprint container by running:
+
+```sh
+docker run \
+--replace \
+--name esdv \
+-d \
+-v $PWD/workspace/workloads:/etc/containers/systemd \
+-v $PWD/workspace/src:/workspace \
+--workdir /workspace \
+quay.io/centos-sig-automotive/autosd:latest
+```
+
+The container will be running in the background and you can enter into it by running:
+
+```sh
+docker exect -it esdv /bin/bash
+```
+
+Enter into the devcontainer and interact with BlueChi:
+
+```sh
+docker exec -it esdv /bin/bash
+bluechictl list-units
+```
+
+## Managing Workloads
+
+This section describes how to deploy and perform administrative's tasks using
+systemd and BlueChi.
+
+### Deploying Applications
+
+BlueChi relies on three components to handle containerized applications:
+
+* systemd
+* Quadlet
+* Podman
+
+Application definitions are stored in `/etc/containers/systemd`. An application
+needs two essential files:
+
+* `{SERVICE_NAME}.kube`: Used by systemd to point to a Kubernetes resource definition
+  containing the workload definition.
+
+  Example of `freyja.kube`:
+    ```kube
+    # https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html
+    [Kube]
+    Yaml=freyja.yml
+
+    # Commented to disable the service to automatically start
+    # [Install]
+    # WantedBy=default.target
+    ```
+* `{SERVICE_NAME}.yaml`: A Kubernetes resource definition (either `v1.Pod` or
+  `apps/v1.Deployment`) that describes the workload.
+
+  Example of `freyja.yml`:
+    ```yaml
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: freyja
+      name: freyja
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: freyja
+      template:
+        metadata:
+          labels:
+            app: freyja
+        spec:
+          hostNetwork: true
+          containers:
+            - name: local
+              image: sdvblueprint.azurecr.io/sdvblueprint/eclipse-freyja/local-with-ibeji:0.1.0
```

Comment:
Consider updating this to the updated path for this image, since the one listed here is private.

Reply:
for reference, the workflow is part of #17